pages/common/bat.md

```diff
@@ -0,0 +1,23 @@
+# bat
+
+> Print and concatenate files. A cat(1) clone with syntax highlighting and Git integration.
+
+- Print the contents of a file to the standard output:
+
+`bat {{file}}`
+
+- Concatenate several files into the target file:
+
+`bat {{file1}} {{file2}} > {{target_file}}`
+
+- Append several files into the target file:
+
+`bat {{file1}} {{file2}} >> {{target_file}}`
+
+- Number all output lines:
+
+`bat -n {{file}}`
+
+- Syntax highlight a json file:
```

Comment:
We should also add `--list-languages` so that the user knows what languages are available.

Reply:
Oh, i missed that :(